lib/mortar/fire_command.rb

```diff
@@ -85,20 +107,31 @@ def resources_output(resources)
     def variables_struct
       return @variables_struct if @variables_struct
 
+      @variables_struct = @configuration.variables(variables_hash)
+
+      @variables_struct
```

Comment:
Unnecessary, the assignment above already returns the same thing. 

The method can also be just the regular memoization type of thing:
`@variables_struct ||= @configuration.variables(variables_hash)`.

Reply:
fixed